### ISSUE TITLE
Insecure flag for CassandraBackup

### ISSUE THREAD

**Issue by an-tex** [Orange-OpenSource/casskop#310]
## Feature Request

**Is your feature request related to a problem? Please describe.**

When using a self-deployed S3 solution like minio the use case arises for using insecure HTTP connections. This currently can't be configured.

**Describe the solution you'd like to see**
https://github.com/instaclustr/icarus allows setting an "insecure: true" flag. This flag could be included in the CassandraBackup CRD which in turn passes it's value to icarus

**Comment by cscetbon** [comment]
@an-tex what about providing a PR ? The change should be minimal, at the CRD, in the controller and a unit test to confirm it’s taken into account. That should be a parameter on both the backup and restore objects.

**Comment by an-tex** [comment]
Thanks for the directions @cscetbon . I've never done anything with go but I'll give it a shot :)

**Comment by an-tex** [comment]
Waiting for https://github.com/instaclustr/instaclustr-icarus-go-client/issues/3

**Comment by cscetbon** [comment]
@an-tex It seems that PR has been merged. It might be a good time to get back to it ;)

**Comment by an-tex** [comment]
Yes but there's not package available, please have a look here https://github.com/instaclustr/instaclustr-icarus-go-client/pull/4#issuecomment-816490030 @cscetbon

**Comment by cscetbon** [comment]
@an-tex I'm not sure why you need a package ? CassKop depends on the GitHub repo directly not on any released package, see https://github.com/Orange-OpenSource/casskop/blob/master/go.mod#L17
